pyomo/contrib/mindtpy/tests/batchdes.py

```diff
@@ -0,0 +1,84 @@
+#  MINLP written by GAMS Convert at 11/10/17 15:35:19
```

Comment:
Please add a reference to the original model.  Since it is a direct translation of another model written in GAMS, we need to know the license under which the original model was released.

Reply:
Solved in the last commit.